src/blockchain/Blockchain.ts

```diff
@@ -47,56 +52,108 @@ export interface IBlockchainApi {
     identity: Identity,
     tx: SubmittableExtrinsic
   ): Promise<SubmittableExtrinsic>
+  submitSignedTx(
+    identity: Identity,
+    tx: SubmittableExtrinsic,
+    opts?: SubscriptionPromiseOptions
+  ): Promise<SubmittableResult>
```

Comment:
again, we have `submitTx` for transactions that need to be (re)signed - is there no way we keep the distinction? I thought it was a nice feature that you didn't have to worry about selecting the correct Identity here

Reply:
Yeah we could have something, that prevents retrying if no identity is given. This wouldn't actually be very complicated as we'd just have to set the retry to something that just doesn't retry.

Reply:
maybe we could have a `retryWith` parameter on the opts object?

Reply:
I set the identity parameter to optional, and if it's not given, then we don't retry

Reply:
nice 👍